backend/blueprints/receipts.py

```
@@ -11,6 +13,39 @@
+        if response.status_code == 200:
+            result = response.json()
+            if result.get('success'):
+                return result.get('url')
+            else:
+                current_app.logger.error(f"Blob upload failed: {result.get('error')}")
+                return None
+        else:
+            current_app.logger.error(f"Blob upload failed with status {response.status_code}: {response.text}")
+            return None
+            
+    except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)